cmake/CMakejucer.cmake

```diff
@@ -331,3 +331,31 @@ function(__version_to_hex version out_hex_value)
   set(${out_hex_value} "${hex_value}" PARENT_SCOPE)
 
 endfunction()
+
+
+function(__four_chars_to_hex value out_hex_value)
+
+  # TODO find a way to make this actually work (";", "[", and "]" are special
+  # characters for CMake)
```

Comment:
@MartyLake: I think I know how to address this TODO, and I will do it before merging this PR.